extensions/azurehybridtoolkit/notebooks/hybridbook/content/hadr/configure-ag.ipynb

```diff
@@ -0,0 +1,143 @@
+{
+    "metadata": {
+        "kernelspec": {
+            "name": "powershell",
+            "display_name": "PowerShell",
+            "language": "powershell"
+        },
+        "language_info": {
+            "name": "powershell",
+            "codemirror_mode": "shell",
+            "mimetype": "text/x-sh",
+            "file_extension": ".ps1"
+        }
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Create and configure Availability Group\n",
+                "\n",
+                "## Description\n",
+                "\n",
+                "Notebook to walk through creating an Availability Group via PowerShell commands"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "ff486dfd-93d4-4a5d-8985-6a352b87ad5e"
+            }
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "Note: before running the steps below, make sure that the following requirements have been satisfied:\n",
+                "\n",
+                "- Verify host instances of SQL Server each resides on a different node of a single failover cluster.\n",
+                "- Verify server instances met the other server-instance prerequisites.\n",
+                "- All the other Always On availability groups requirements are met and that all recommendations are considered.\n",
+                "- Primary server must have at least one user made database in order to create secondary replicas.\n",
+                "- Domain account must have sysadmin fixed server role, andeither CREATE AVAILABILITY GROUP server permission, ALTER ANY AVAILABILITY GROUP permission, or CONTROL SERVER permission.\n",
+                "\n",
+                "For more information on Always On Prerequisites see [here](https://docs.microsoft.com/en-us/sql/database-engine/availability-groups/windows/prereqs-restrictions-recommendations-always-on-availability?view=sql-server-ver15).\n",
+                "\n",
+                "SQL Server Failover Cluster Instances (FCI) do not support automatic failover by availability groups, so any availability replica hosted by an FCI can only be configured for manual failover. Here is a listing of variables that will be used to create the availability group:\n",
+                "\n",
+                "| Line # | Name | Example | Description |\n",
```

Comment:
Consider changing the "column" order to: "Line #", "Name", "Description", and "Example" as I've done in other notebooks. You can choose to ignore this comment and make no changes if it's too much of a hassle.